modules/alloy/src/test/ts/atomic/position/view/BounderCalcRepositionTest.ts

```diff
@@ -6,18 +6,19 @@ import * as Bounder from 'ephox/alloy/positioning/view/Bounder';
 
 UnitTest.test('BounderCalcRepositionTest', () => {
 
-  const nonZeroArb = Jsc.integer(10, 1000);
-  const zeroableArb = Jsc.integer(0, 1000);
+  const maxBounds = 2000;
+  const minBounds  = 0;
+  const zeroableArb = Jsc.integer(minBounds, maxBounds);
 
   const arbTestCase = Jsc.bless({
-    generator: zeroableArb.generator.flatMap((newX: number) => {
-      return zeroableArb.generator.flatMap((newY: number) => {
-        return nonZeroArb.generator.flatMap((width: number) => {
-          return nonZeroArb.generator.flatMap((height: number) => {
-            return zeroableArb.generator.flatMap((boundsX: number) => {
-              return zeroableArb.generator.flatMap((boundsY: number) => {
-                return zeroableArb.generator.flatMap((boundsW: number) => {
-                  return zeroableArb.generator.map((boundsH: number) => {
+    generator: zeroableArb.generator.flatMap((boundsX: number) => {
```

Comment:
Is there a lot of jsverify still around that we want to upgrade to fastcheck (or whatever the more strongly-typed library was called)

Reply:
We really don't have time to be rewriting entire tests in `fastcheck` at this time. I'll take a look and if it can be done quickly I'll make the change, otherwise this will have to stay as is.

Reply:
Had a look and due to the differences, it's not easy to port across.

Reply:
yeah `bless` in particular is a bit of a challenge, I think there's an entire separate repo with a library to help migrate blessed generators to fast-check